x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/type/Schema.java

```diff
@@ -16,21 +16,29 @@
 import java.util.stream.StreamSupport;
 
 import static java.util.Collections.emptyList;
+import static java.util.Collections.nCopies;
+import static org.elasticsearch.common.Strings.hasText;
 
 public class Schema implements Iterable<Schema.Entry> {
```

Comment:
Since this is in QL, *if* the Schema needs to be extended, it should be in SQL.

Reply:
Ah, right, thanks.